examples/migrate_cluster_to_advanced_cluster/module_maintainer/v1/variables.tf

```
@@ -0,0 +1,68 @@
+variable "project_id" {
+  type = string
+}
+
+variable "cluster_name" {
+  type = string
+}
+
+variable "cluster_type" {
+  type = string
+
+  validation {
+    condition     = contains(["REPLICASET", "SHARDED", "GEOSHARDED"], var.cluster_type)
```

should we have these conditions here or pass to Atlas and let it fail so we don't duplicate checks here in TF?

---

Removed in bfb05c04c0aa7d61996fb958513a4bb9f4cff4a7